packages/core/src/types.ts

```diff
@@ -67,6 +67,11 @@ export type UserOperationOverrides = Partial<{
    * one user operation for your account in a bundle
    */
   nonceKey: bigint;
+  /**
+   * The signature override is used in special cases when custom signature middleware is used
+   * This middleware allows you to provide context to your custom signature middleware
+   */
+  signature: UserOperationStruct["signature"];
```

Comment:
the thing with multisig plugin though is that you have to  encode calls in `executeWithSessionKey` as the UO calldata. so for that one you wouldn't be using signature overrides.

I agree though this isn't super clean but at this point I can't think of a better way to do it in the multisig case. multisig requires the context of the previous signatures to work

Reply:
given that this is actually meant to be context and not override, I'm wondering if we need to support a `context` parameter sendUserOperation instead of using `overrides` for this since it's not exactly an override